toolchain/sem_ir/inst_kind.h

```diff
@@ -43,45 +43,47 @@ enum class InstConstantKind : int8_t {
   // concrete constant, the result is non-constant. This is the default.
   Indirect,
-  // This instruction may define a symbolic constant, depending on its operands,
-  // but never a concrete constant. For example, a `Call` instruction can define
-  // a symbolic constant but never a concrete constant. The instruction may have
-  // a concrete constant value of a different kind.
+  // This instruction may represent a symbolic constant, depending on its
+  // operands, but never a concrete constant. For example, a `Call` instruction
+  // can represent a symbolic constant but never a concrete constant. The
+  // instruction may have a concrete constant value of a different kind.
   SymbolicOnly,
   // This instruction is a metaprogramming or template instantiation action that
-  // generates an instruction. Like `SymbolicOnly`, it may define a symbolic
-  // constant, depending on its operands, but never defines a concrete constant.
-  // The instruction may have a concrete constant value that is a generated
-  // instruction. Constant evaluation support for types with this constant kind
-  // is provided automatically, by calling `PerformDelayedAction`.
+  // generates an instruction. Like `SymbolicOnly`, it may represent a symbolic
+  // constant, depending on its operands, but never represents a concrete
+  // constant. The instruction may have a concrete constant value that is a
+  // generated instruction. Constant evaluation support for types with this
+  // constant kind is provided automatically, by calling `PerformDelayedAction`.
   InstAction,
-  // This instruction can define a symbolic or concrete constant, but might not
-  // have a constant value, might have a constant value that is not defined by
-  // itself, or might result in a compile-time error, depending on its operands.
-  // For example, `ArrayType` is a compile-time constant if its operands are
-  // constant and its array bound is within a valid range.
+  // This instruction can represent a symbolic or concrete constant, but might
+  // not have a constant value, might have a constant value that is not defined
+  // by itself, or might result in a compile-time error, depending on its
+  // operands. For example, `ArrayType` is a compile-time constant if its
+  // operands are constant and its array bound is within a valid range.
   Conditional,
-  // This instruction defines a symbolic or concrete constant whenever its
+  // This instruction represents a symbolic or concrete constant whenever its
   // operands are constant. Otherwise, it is non-constant. For example, a
-  // `TupleValue` defines a constant if and only if its operands are constants.
-  // Constant evaluation support for types with this constant kind is provided
-  // automatically.
+  // `TupleValue` represents a constant if and only if its operands represent
+  // constants. Constant evaluation support for types with this constant kind is
+  // provided automatically.
```

Comment:
I think the pre-existing description here is wrong. How about this:
```suggestion
  // This instruction can represent a symbolic or concrete constant, and has a
  // constant value of the same kind whenever its operands are constant.
  // Otherwise, it is non-constant. For example, a `TupleValue` has a constant
  // value if and only if its operands have constant values, and its constant
  // value is always represented by a `TupleValue` instruction. Constant
  // evaluation support for types with this constant kind is provided
  // automatically.
```

Reply:
As noted below, I now think all three versions are wrong.